client/src/pages/ProjectsPage.js

```diff
@@ -26,18 +50,49 @@ class Projects extends Component {
                                     </button>
                                 </div>
                                 <div className="modal-body">
-                                   FORM GOES HERE 
+                                    <form>
+                                        <div className="form-group">
+                                            <label for="inputTitle">Title</label>
+                                            <input 
+                                                // value={this.state.title}
+                                                // onSubmit={this.handleSubmit.bind(this)}
```

Comment:
Good practice to keep comments cleaned out even during development. Helps so you don't miss deleting them later.